### PR TITLE
Update NCEPU.md

### PR DESCRIPTION
**Pull request by fluffyzhizi**

### CODE REVIEW

File: docs/campus/NCEPU.md [hunk 1]
```diff
@@ -13,11 +13,11 @@ author: 植梓 等
 
 ## 地理位置及交通
 
-学校位于 [北京市昌平区北农路 2 号](https://amap.com/place/B000A07C0B)。地处城郊，想要去城区基本就是昌 25 路公交转地铁；到复兴医院大概要两个小时，其中步行距离接近两公里，不过可以租用共享单车。
+学校位于 [北京市昌平区北农路 2 号](https://amap.com/place/B000A07C0B)。地处城郊，想要去城区基本就是昌 25 路公交转地铁；好处是到回龙观非常方便，从西南门出校骑车只要十五分钟。
 
 ## 军训与体育
 
-军训总的来说比较宽松，身体不适等各种理由都可以申请休息，但是军训期间不能离开操场。
+军训总的来说比较宽松，身体不适等各种理由都可以申请休息，但是军训期间不能离开操场,建议参选宣传部，可以免除大多数训练。
```
Comment (Cryolitia) [inline docs/campus/NCEPU.md:20]:
```suggestion
军训总的来说比较宽松，身体不适等各种理由都可以申请休息，但是军训期间不能离开操场，建议参选宣传部，可以免除大多数训练。
```

什么全角半角混用

File: docs/campus/NCEPU.md [hunk 2]
```diff
@@ -33,13 +33,13 @@ author: 植梓 等
 
 ## 医疗资源
 
-校医院属于一级甲等医院；设有妇科和医学影像科、药剂科，但是据去过的同学说实际医疗效果几乎没有；不清楚是否能查激素六项。
+校医院属于一级甲等医院；设有妇科和医学影像科、药剂科，但是据去过的同学说实际医疗效果几乎没有；经过询问无法检测激素六项，也买不到补子和凝胶，但是开转诊单并不会被卡。
 
 ## LGBT 友善梗概
 
-对于 MtF 接受度，因为疫情的原因大部分时间都是在寝室上网课，目前我还是处于半出柜的状态。第一次女装的话几乎整层楼的人都会围过来起哄，当然也有个人素质的原因，多了之后习惯了接受度也挺高。目前学校有长期女装的学生（包括上课期间）并未在学业方面遇到阻碍。由于疫情原因基本在上网课，目前没有出柜的 MtF（也有可能是我不知道），据研一学长说他的朋友前段时间刚做了 SRS 手术，也有好几个伪娘在咨询关于 SRS 手术的问题，大概学校内的 MtF 都比较谨慎（我也是）。
+目前学校已知在校仅我一个mtf，另有一位ftm，学校对女装接受度挺高的。目前学校有长期女装的学生（包括上课期间）并未在学业方面遇到阻碍。由于疫情原因基本在上网课，目前没有出柜的 MtF（也有可能是我不知道），据研一学长说他的朋友前段时间刚做了 SRS 手术，也有好几个伪娘在咨询关于 SRS 手术的问题。
```
Comment (Cryolitia) [inline docs/campus/NCEPU.md:40]:
```suggestion
目前学校已知在校仅我一个 MtF，另有一位 FtM，学校对女装接受度挺高的。目前学校有长期女装的学生（包括上课期间）并未在学业方面遇到阻碍。由于疫情原因基本在上网课，目前没有出柜的 MtF（也有可能是我不知道），据研一学长说他的朋友前段时间刚做了 SRS 手术，也有好几个伪娘在咨询关于 SRS 手术的问题。
```

File: docs/campus/NCEPU.md [hunk 3]
```diff
@@ -33,13 +33,13 @@ author: 植梓 等
...
 
-经过一段时间的试探可以确定动漫社对于 MtF 的接受度较高，（还是二刺猿好人多啊 qwq）学校动漫社分布较杂，就目前体验而言方舟圈的环境对 MtF 最友善。cos 群有不少摄影师社牛属性拉满线下拉着就打招呼挺尴尬的。
+目前已经在学校方舟群出柜，，接受度挺高的，现在基本都叫我学姐，（还是二刺猿好人多啊 qwq）学校动漫社分布较杂，就目前体验而言方舟圈的环境对 MtF 最友善。cos 群有不少摄影师社牛属性拉满线下拉着就打招呼挺尴尬的。
```
Comment (Cryolitia) [inline docs/campus/NCEPU.md:42]: 羡慕死了

**Comment by BeiyanYunyi** [comment]
坏了我怎么点到那个 sign off and commit suggestion 了

**Review by Cryolitia — APPROVED** [review]

**Comment by Cryolitia** [comment]
@BeiyanYunyi 坏了你怎么点到那个 sign off and commit suggestion 了

**Review by HalfSweet — APPROVED** [review]

**Review by Cryolitia — APPROVED** [review]

**Pull request merged**